### ISSUE TITLE
Is it cool if I start opening some pull requests here?

### ISSUE THREAD

**Issue by StoneCypher** [mckenzieasimone/hairstylistapp#2]
If it's a small thing, sometimes the other programmer will just handle it, and offer it handled, instead of saying "you should go do this."

By example, if I found a typo or a grammar error in a README, I wouldn't bother to tell someone about it.  I'd just offer them a fix, and explain why in the comment.

Alternately, when you're teaching, or leading a team at work, this is also a good method.

Finally, when two people are working together on different parts of the same system.  By example, my friend and I are currently working on a thing.  I'm doing one piece, he's doing another, they don't overlap a lot, but we check each other's work on the way through.  Second set of eyes, style of thing.

**Comment by mckenzieasimone** [comment]
Yes :) I'm going to get a basic homepage up just to have something.
